_posts/2017-08-17-grpc-flatbuffers.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: post
+title: Announcing out of the box support for gRPC in the Flatbuffers serializaton library.
+published: true
+permalink: blog/flatbuffers
+author: Wouter van Oortmerssen
+company: Google
+company-link: https://www.google.com
+---
+
+The recent release of Flatbuffers [version 1.7](https://github.com/google/flatbuffers/releases) introduced truly zero-copy support for gRPC out of the box. 
+
+[Flatbuffers](https://google.github.io/flatbuffers/) is a serialization library that allows you to access serialized data without first unpacking it or allocating any 
+additional data structures. It was originally designed for games and other resource constrained applications, but is now finding use
+within many teams at Google, and outside (eg: Facebook, Netflix).
+
+<!--more-->
+
+Flatbuffers enables maximum throughput by directly using gRPC's slice buffers with zero-copy for the common use cases. An incoming rpc may be processed directly from gRPCs internal buffers, and you constructing new message will write directly to internal buffers without intermediate steps.
```

Comment:
with zero-copy for common use cases. An incoming RPC can be processed directly from gRPC's internal buffers, while constructing a new message will write directly to these buffers without intermediate steps.